.circleci/config.yml

```diff
@@ -175,15 +175,32 @@ jobs:
       - deploy: #deploy step is important to prevent triggering N builds
           name: Publish next packages
           command: >-
-            yarn lerna:prerelease --yes --no-push &&
```

Comment:
We can remove all of the `Configure Git` from this job so we just have it for the `commit_next` job.

Reply:
Thanks, done

Reply:
React component tests failing - https://circleci.com/gh/Royal-Navy/design-system/35365

Reply:
If you rebase this the test should now pass.

Reply:
thanks, all good now